ios/FluentUI/List/FluentList.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// Fluent specific list style enum
+public enum FluentListStyle {
+    case plain
```

Comment:
Do we actually plan to support all four of these styles?

Reply:
Went ahead and removed `.grouped`.